### PR TITLE
in case of new RTM message send it to OEM as UnrecognizedRtmMessage

### PR DESCRIPTION
**Pull request by skynet67**

### CODE REVIEW

File: fitpay/src/main/java/com/fitpay/android/webview/events/UnrecognizedRtmMessage.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.fitpay.android.webview.events;
+
+/**
+ * Unrecognized {@link RtmMessage}
+ */
+
+public class UnrecognizedRtmMessage extends RtmMessage {
```
Comment (ssteveli) [inline fitpay/src/main/java/com/fitpay/android/webview/events/UnrecognizedRtmMessage.java]: So I'm just not thrilled with this naming convention.  My desire is to allow the webapp to just add RTM messages at will without having to update the SDK.  While an OEM could listen for "UnrecognizedRtmMessage", that just doesn't feel right if it's a use case they need to support.  I just think `RtmMessage` should get spit out and they can generically listen for that, the introspect the paylaod to determine if it's a message they're interested in.

**Comment by Polar-Beard** [comment]
@Clark73 are you still waiting for changes on this?

File: fitpay/src/main/java/com/fitpay/android/webview/impl/parser/RtmParser.java [hunk 2]
```diff
@@ -18,6 +20,7 @@ public RtmParser(WebViewCommunicatorImpl impl) {
 
     public void parseMessage(RtmMessage msg) {
         FPLog.d(TAG, String.format("Unrecognized RTM message of type %s. Skipping.", msg.getType()));
+        RxBus.getInstance().post(new UnrecognizedRtmMessage(msg));
```
Comment (Polar-Beard) [inline fitpay/src/main/java/com/fitpay/android/webview/impl/parser/RtmParser.java]: I'm not sure why this is being posted. Didn't we talk about just ignoring it and letting the OEM's filter based on the type of RtmMessage they're interested in?

Comment (skynet67) [reply fitpay/src/main/java/com/fitpay/android/webview/impl/parser/RtmParser.java]: wait a few sec, will push an update

**Comment by Polar-Beard** [comment]
The travis test that is failing runs fine locally, so I'm going to go ahead and merge this branch.

**Pull request merged**